plugins/CoreUpdater/Updater.php

```diff
@@ -138,9 +139,45 @@ public function updatePiwik($https = true)
             throw new UpdaterException($e, $messages);
         }
 
+        $partTwoUrl = Url::getCurrentUrlWithoutQueryString() . Url::getCurrentQueryStringWithParametersModified([
```

Comment:
Shouldn't the complete block within `if (Marketplace::isMarketplaceEnabled())` now be unneeded in the lines above?

Reply:
👍 removed it